### PR TITLE
#24 Fix email regex

### PR DESCRIPTION
**Pull request by vmdhhh**
I think this should be enough for a normal email check.
Please let me know your thoughts.

### CODE REVIEW

File: mongoengine_migrate/fields/converters/common.py [hunk 1]
```diff
@@ -385,7 +385,7 @@ def by_path(ctx: ByPathContext):
     to_string(updater)
 
     email_regex = re.compile(
-        r"\A.*\Z",  # TODO: insert email validation regex here
+        r"\A[^\W][A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}\Z",
```
[the remark this answers is not in the document]
Comment (vmdhhh) [reply mongoengine_migrate/fields/converters/common.py]: @bdragon300  I have made the suggested changes. Let me know if it's ok.

[the remark this answers is not in the document]
Comment (bdragon300) [reply mongoengine_migrate/fields/converters/common.py]: @vmdhhh `A-Z` in domain name (after `@`) should also be changed to `\p{L}` to support IDN domains.

File: mongoengine_migrate/fields/converters/common.py [hunk 2]
```diff
@@ -379,15 +379,12 @@ def by_path(ctx: ByPathContext):
 
 def to_email_string(updater: DocumentUpdater):
     def by_path(ctx: ByPathContext):
-        fltr = {ctx.filter_dotpath: {'$not': email_regex, '$ne': None}, **ctx.extra_filter}
+        email_regex = r"\A[^\W][A-Z0-9._%+-]+@[A-Z0-9.-]+\.\p{L}+\Z"
```
Comment (bdragon300) [inline mongoengine_migrate/fields/converters/common.py:382]: @vmdhhh A-Z in domain name (after @) should also be changed to \p{L} to support IDN domains.

**Review by bdragon300 — CHANGES_REQUESTED** [review]

**Review by bdragon300 — APPROVED** [review]
Looks good. Merged, thanks.

**Pull request merged**